crawl-ref/source/spl-summoning.cc

```diff
@@ -1890,30 +1890,30 @@ spret_type cast_animate_skeleton(god_type god, bool fail)
             break;
     }
 
-    return SPRET_SUCCESS;
+    return spret::success;
 }
 
-spret_type cast_animate_dead(int pow, god_type god, bool fail)
+spret cast_animate_dead(int pow, god_type god, bool fail)
 {
     fail_check();
     canned_msg(MSG_CALL_DEAD);
 
     if (!animate_dead(&you, pow + 1, BEH_FRIENDLY, MHITYOU, &you, "", god))
         canned_msg(MSG_NOTHING_HAPPENS);
 
-    return SPRET_SUCCESS;
+    return spret::success;
 }
 
 /**
  * Have the player cast simulacrum.
  *
  * @param pow The spell power.
  * @param god The god casting the spell.
- * @param fail If true, return SPRET_FAIL unless the spell is aborted.
- * @returns SPRET_ABORT if no viable corpse was at the player's location,
- *          otherwise SPRET_TRUE or SPRET_FAIL based on fail.
+ * @param fail If true, return spret::fail unless the spell is aborted.
+ * @returns spret::abort if no viable corpse was at the player's location,
+ *          otherwise SPRET_TRUE or spret::fail based on fail.
```

Comment:
Comment here has one unconverted enum value.

Reply:
Right, because SPRET_TRUE was never an enum value in the first place, but yea you're right I should've run `git grep SPRET` to check.